### DISCUSSION TITLE
Github仓库赞助链接失效

### DISCUSSION THREAD

**Discussion by RhoPaper** [travellings-link/travellings#2697] [发展建议]
开往 Github仓库右侧赞助的爱发电链接失效了，好像是之前爱发电改域名的缘故……
应该由现在的https://afdian.net/a/travellings改成https://ifdian.net/a/travellings
![现在的链接](https://github.com/user-attachments/assets/244b1469-ea67-4ea1-86c1-de28262cafea)
![无法访问](https://github.com/user-attachments/assets/6de37535-9461-431e-80e0-d64ec19f5fd8)

**Comment by gxres042** [comment]
已修改
